infra.bs

```diff
@@ -679,6 +679,22 @@ where <a>UTF-8 encode</a> comes into play.
 
 <hr>
 
+<p>A <a>string</a> <var>a</var> <dfn export for=string lt="is|identical to">is</dfn> or is 
+<a for=string>identical to</a> a <a>string</a> <var>b</var> if it consists of the same sequence of 
+<a>code units</a>.
+
+<p>Except where otherwise stated, all string comparisons use <a for=string>is</a>.
+
+<div class=note>
+
```

Comment:
This newline can be removed and we should use for=string in the "identical to" mentions below to reduce the amount of refactoring we need to do in the future. Looks good to me otherwise.

@domenic thoughts?

Reply:
This should actually be a `<p class="note">`. I'm happy to push a fixup commit for that (and the other issue you mentioned).

Reply:
I tried to follow the local style :-). Thanks for the feedback.